sepia-fog-images/config/definitions/sepia-fog-images.yml

```diff
@@ -15,10 +15,14 @@
           artifact-days-to-keep: -1
           artifact-num-to-keep: -1
 
+    # Run every Sunday at noon
+    triggers:
+      - timed: "0 12 * * 0"
+
     parameters:
       - string:
           name: DISTROS
-          default: "ubuntu_14.04 ubuntu_16.04 centos_7.5 rhel_7.5"
+          default: "ubuntu_18.04 centos_8.1 rhel_8.1"
```

Comment:
Can we have that for openSUSE images, too?

Reply:
I need some help from Suse with the autoinstallation.  It works from start to finish automatically on 15.1 but 15.2 stops in the installation and asks for the URL of the ISO repo even though it is passed as a kernel option.

Reply:
are you using autoyast? is there more information what you are doing?

Reply:
Sorry, yes, I started to gather this info and got pulled away.

Here is our autoyast XML: https://github.com/ceph/ceph-cm-ansible/blob/master/roles/cobbler/templates/kickstarts/cephlab_opensuse_leap.xml

Here is what is being passed as a kernel parameter to the installation: `install=http://172.21.0.11/cblr/links/openSUSE-15.1-x86_64/` (of course the URL changes to 15.2 if the distro is 15.2)

Installation completes successfully in 15.1 but the installer stops and asks for the link above during 15.2 autoinstall.

Reply:
Is there anything in the logs maybe? Or is it possible to access the system?

Reply:
As far as I can tell, there is no way to check logs from within YaST.

On 15.1, the installer runs from start to finish.  On 15.2, it stops at `please make sure your installation media is available`

Reply:
You're welcome to ssh to smithi166 and dig through `/var/log/YaST2`

Reply:
I'll try to have a look. sorry for not responding anymore....